api/schemas/source.json

```diff
@@ -51,6 +51,21 @@
         },
         "tags": {
             "$ref": "tags.json"
+        },
+        "source_collection": {
+            "type": "array",
+            "description": "List of Sources that are collected together by this Source. This attribute is intended to be read-only. Implementations SHOULD ignore this if given in a PUT request, and instead manage it internally. Source collections can be inferred from Flow collection definitions.",
```

Comment:
How do I construct a custom group in this model? Say I was delivered a package of video, English audio and English subtitles, and I then commissioned Welsh subtitles. Can I construct a Source that refers to video, English audio, Welsh subs without generating an asset containing all three (e.g. because I want to create the asset later)? Do I just create an empty Flow collecting the right Flows together, and then "make it real" at the time I actually need it?

Reply:
The application note (to follow) will cover this, but essentially yes, what you propose is more or less how you'd achieve this.

Reply:
I think it seems nicer to be able to populate a Source structure without resorting to a side-effect of creating empty / placeholder Flows. The API has an endpoint to PUT a Source and so it seems reasonable to allow the same for a multi-essence source? That still requires a multi-essence Flow to be created to specify which flows were grouped.

I suppose part of this relates to what we decide (have decided, don't remember) happens if a Flow is deleted; is the Source deleted as well if no more Flows reference it? I.e. do we consider the Sources to be a view only on Flows in the TAMS.